Dockerfile

```diff
@@ -25,6 +25,24 @@ COPY . /context
 WORKDIR /context
 RUN touch dev-requirements.txt && pip install --upgrade pip && pip install -c dev-requirements.txt .
 
+FROM build AS debug
+
+# Set origin to use ssh
+RUN git remote set-url origin git@github.com:{{github_org}}/DiamondLightSource/blueapi.git
```

Comment:
```suggestion
RUN git remote set-url origin git@github.com:diamondlightsource/DiamondLightSource/blueapi.git
```

Reply:
you know it's already shorter than line 26 right?